Travis CI is failing with the following issue...

> Warning, treated as error:
> /home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pypahdb/decomposer.py:docstring of pypahdb.decomposer.decomposer.charge:4:Field list ends without a blank line; unexpected unindent.
> make: *** [html] Error 2
> make: Leaving directory `/home/travis/build/PAHdb/pyPAHdb/docs'
> The command "make -C docs html" exited with 2.

Not able to reproduce on my end when running "make -C docs html". However, I did have to create the "docs/source/_static" folder locally. I wonder if this is the issue here, despite the error message not really saying as such.

--- a/docs/source/index.rst
+++ b/docs/source/index.rst
@@ -14,6 +14,7 @@ Welcome to pypahdb's documentation!
    installation
    usage
    release-history
+   pypahdb
    modules
 
 Indices and tables

--- a/docs/source/index.rst
+++ b/docs/source/index.rst
@@ -14,7 +14,6 @@ Welcome to pypahdb's documentation!
    installation
    usage
    release-history
-   pypahdb
    modules
 
 Indices and tables

--- a/docs/source/pypahdb.rst
+++ b/docs/source/pypahdb.rst
@@ -26,6 +26,7 @@ pypahdb\.observation module
     :members:
     :undoc-members:
     :show-inheritance:
+    :noindex:
 
 pypahdb\.spectrum module
 ------------------------

--- a/pypahdb/decomposer.py
+++ b/pypahdb/decomposer.py
@@ -70,13 +70,11 @@ class decomposer(object):
        spectrum: The spectrum to fit and decompose.
     """
     def __init__(self, spectrum):
-        """
-        Construct a decomposer object.
+        """Construct a decomposer object.
 
-        :param spectrum: The spectrum to fit and decompose
-        :return: returns None
+        Args:
+            spectrum (object): The spectrum to fit and decompose
         """
-
         self._yfit = None
         self._ionized_fraction = None
         self._large_fraction = None
@@ -135,10 +133,10 @@ class decomposer(object):
         self.norm = np.reshape(self.norm, (ordd.shape[2], ordd.shape[1])).T
 
     def _fit(self):
-        """
-        Return the fit.
+        """Return the fit.
 
-        :return: returns array
+        Returns:
+            self._yfit (ndarray): PAHdb total fit.
         """
 
         # Lazy Instantiation
@@ -177,10 +175,10 @@ class decomposer(object):
         return self._yfit
 
     def _get_ionized_fraction(self):
-        """
-        Return the ionized fraction.
+        """Return the ionized fraction.
 
-        :return: returns array
+        Returns:
+            self._ionized_fraction (ndarray): Ion fraction of fit.
         """
 
         # Lazy Instantiation
@@ -202,10 +200,10 @@ class decomposer(object):
         return self._ionized_fraction
 
     def _get_large_fraction(self):
-        """
-        Return the large fraction.
+        """Return the large fraction.
 
-        :return: returns array
+        Returns:
+            self._large_fraction (ndarray): Large fraction of fit.
         """
 
         # Lazy Instantiation
@@ -229,8 +227,9 @@ class decomposer(object):
     def _get_charge(self):
         """Return the spectral charge breakdown.
 
-        :return: returns associative array with keys
-        'anion', 'neutral' and 'cation'
+        Returns:
+            self._charge (ndarray): Associative array with keys
+                'anion', 'neutral' and 'cation'
         """
 
         # Lazy Instantiation
@@ -313,7 +312,9 @@ class decomposer(object):
     def _get_size(self):
         """Return the spectral size breakdown.
 
-        :return: returns associative array with keys 'large' and 'small'
+        Returns:
+            self._size (ndarray): Associative array with keys
+                'large', 'small'
         """
 
         # Lazy Instantiation

